prisma/schema.prisma

```diff
@@ -27,4 +28,10 @@ model User {
   events        Event[]
 }
 
-
+model Atendees {
+  id            Int       @id @default(autoincrement())
+  name          String?
+  event_name    Event?    @relation(fields: [eventId], references: [id])
```

Comment:
can you call this `event` instead?

let's stay by this convention: "if you have a relationship let's call it by the table name"

Reply:
having a different name should be the exception